### PR TITLE
Misc improvements

### PR DESCRIPTION
**Pull request by JeremyRH**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -8,7 +8,9 @@
           "last 2 Firefox versions",
           "last 2 Edge versions",
           "last 2 Safari versions"
-        ]
+        ],
+        // For uglifyjs
+        "uglify": true
```
Comment (samhh) [inline .babelrc]: Is there any documentation for this key?

File: src/modules/fullscreen.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+const requestFullscreen = (element) => {
+	if (element.requestFullscreen) {
+		element.requestFullscreen()
+	} else if (element.webkitRequestFullscreen) {
+		element.webkitRequestFullscreen()
+	} else if (element.mozRequestFullScreen) {
+		element.mozRequestFullScreen()
+	} else if (element.msRequestFullscreen) {
+		element.msRequestFullscreen()
```
Comment (samhh) [inline src/modules/fullscreen.js]: IE prefixes can be removed, we're only supporting recent Edge and AFAIK is unprefixed.

File: src/modules/fullscreen.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+	if (document.exitFullscreen) {
+		document.exitFullscreen()
+	} else if (document.webkitExitFullscreen) {
+		document.webkitExitFullscreen()
+	} else if (document.mozCancelFullScreen) {
+		document.mozCancelFullScreen()
+	}
+}
+
+const fullscreenElement = () => {
```
Comment (samhh) [inline src/modules/fullscreen.js]: IIRC optionally you can drop these braces and still use 2 lines for readability:

```javascript
const fullscreenElement = () =>
    document.fullscreenElement || document.webkitFullscreenElement || document.mozFullScreenElement
```

**Review by samhh — CHANGES_REQUESTED** [review]
See comments.

File: .babelrc (hunk 1 above)
Comment (JeremyRH) [reply .babelrc]: You can see the note here that `targets.uglify` is deprecated and will be removed in the next major version: https://github.com/babel/babel-preset-env/blob/84e12fcb3e929ca3f5edd65c3b03221a70f11881/README.md#forcealltransforms

If we are not using uglify to minify, we should remove it from our build step.

File: src/modules/fullscreen.js (hunk 3 above)
Comment (JeremyRH) [reply src/modules/fullscreen.js]: Yeah that seems like a good change. Can we enforce this with an eslint rule?

**Review by samhh — APPROVED** [review]

**Pull request merged**